apps/namadillo/src/App/Ibc/IbcTransfer.tsx

```diff
@@ -115,7 +116,8 @@ export const IbcTransfer: React.FC = () => {
         },
       });
       setCurrentStep(2);
-    } catch {
+    } catch (err) {
+      setGeneralErrorMessage(err + "");
```

Comment:
I'm not sure if this is a valid case, but do we also need to unset `generalErrorMessage` when the submission is successful? I think we display a different page on success, so maybe not, but I'm not sure.